crates/floresta-electrum/src/electrum_protocol.rs

```diff
@@ -570,6 +570,8 @@ impl<Blockchain: BlockchainInterface> ElectrumServer<Blockchain> {
     async fn rescan_with_block_filters(
         &mut self,
         cfilters: Arc<NetworkFilters<FlatFiltersStore>>,
+        start_height: Option<u32>,
+        stop_height: Option<u32>,
```

Comment:
Is it needed tho? We need to find a way to compute those. In the current form, these parameters are useless.